chapters/research-design.tex

```diff
@@ -1,288 +1,593 @@
+will not be as good at predicting outcomes or fitting the data as other models.
+Additionally, when control variables or other variables are used in estimation,
+there is no guarantee that those parameters are marginal effects.
+They can only be interpreted as correlative averages,
+unless the experimenter has additional sources of identification for them.
+The models you will construct and estimate are intended to do exactly one thing:
+to express the intention of your project's research design,
+and to accurately estimate the effect of the treatment it is evaluating.
+In other words, these models tell the story of the research design
+in a way that clarifies the exact comparison being made between control and treatment.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Experimental and quasi-experimental research designs}
 
 Experimental research designs explicitly allow the research team
-to change the condition of the populations being studied,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Experimental_Methods}}
-in the form of NGO programs, government regulations,
+to change the condition of the populations being studied,\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Experimental_Methods}}
+often in the form of NGO programs, government regulations,
 information campaigns, and many more types of interventions.\cite{banerjee2009experimental}
-The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
-(Not everyone agrees this is the best way to do research.\sidenote{\url{https://www.nber.org/papers/w14690.pdf}})
-\index{randomized control trial}
-There treatment and control groups are drawn from the same underlying population
-so that the strong condition of statistical equality
-in the absence of the experiment can be assumed.
-Three RCT-based methods are discussed here:
-\textbf{cross-sectional randomization} (``endline-only'' studies),
-\textbf{difference-in-difference} (``panel-data'' studies),
-and \textbf{regression discontinuity} (``cutoff'' studies).
-
-\subsection{Cross-sectional RCTs}
-
-\textbf{Cross-sectional RCTs} are the simplest possible study design:
+The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
+  \index{randomized control trials}
+In randomized control trials, the control group is randomized --
+that is, from an eligible population,
+a random subset of units are not given access to the treatment,
+so that they may serve as a counterfactual for those who are.
+A randomized control group, intuitively, is meant to represent
+how things would have turned out for the treated group
+if they had not been treated, and it is particularly effective at doing so
+as evidenced by its broad credibility in fields ranging from clinical medicine to development.
```

Comment:
really??

Reply:
i don't think a citation is necessary here

Reply:
I agree that RCT has broad credibility in many fields, but in a quick google search, I couldn't find a  source that shows that RCTs are more likely to be published in any field. I also found a bunch of articles and paper saying that RCTs are popular in many fields, but they all seem to rely on the assumption that this is an undisputed fact rather than showing any data to corroborate that. It may be asking too much of this book to do so when no one elses seems to do it, but I'd like to see some numbers.

Reply:
moved to issue #335